docs/credentials-with-undefined-terms.json

```diff
@@ -52,8 +56,8 @@
     "count": 2
   },
   {
-    "type": "OrganicCertificationCertificate",
-    "count": 0
+    "type": "OrganicCertificateCredential",
```

Comment:
why does this change?

Reply:
```suggestion
    "type": "OrganicCertificationCredential",
```

Reply:
this was changes automatically after running `cd packages/traceability-schemas && npm test` locally.

Reply:
Autogen file, should open issue if the name is a problem

Reply:
Moving discussion to: https://github.com/w3c-ccg/traceability-vocab/issues/616